pkg/util/constants/constants.go

```diff
@@ -27,6 +27,10 @@ const (
 	OperatorRoot   = "/var/tmp/etcd-operator"
 	BackupMountDir = "/var/etcd-backup"
 
-	EnvOperatorPodName      = "MY_POD_NAME"
-	EnvOperatorPodNamespace = "MY_POD_NAMESPACE"
+	EnvOperatorPodName            = "MY_POD_NAME"
+	EnvOperatorPodNamespace       = "MY_POD_NAMESPACE"
+	EnvRestoreOperatorServiceName = "SERVICE_ADDR"
```

Comment:
remove this `EnvRestoreOperatorServiceName`

Reply:
oops, bad merged